code/game/objects/items/storage/firstaid.dm

```diff
@@ -320,3 +320,46 @@
 /obj/item/storage/pill_bottle/penis_enlargement/PopulateContents()
 	for(var/i in 1 to 7)
 		new /obj/item/reagent_containers/pill/penis_enlargement(src)
+
+/////////////
+//Organ Box//
+/////////////
+
+/obj/item/storage/belt/organbox
+	name = "Organ Storge"
+	desc = "A compact box that helps hold massive amounts of implants, organs, and some tools. Has a belt clip for easy carrying"
+	w_class = WEIGHT_CLASS_BULKY
+	icon = 'icons/obj/mysterybox.dmi'
+	icon_state = "organbox_open"
+	lefthand_file = 'icons/mob/inhands/equipment/medical_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
+	throw_speed = 1
+	throw_range = 1
+
+/obj/item/storage/belt/organbox/ComponentInitialize()
+	. = ..()
+	GET_COMPONENT(STR, /datum/component/storage)
+	STR.max_items = 16
+	STR.max_w_class = WEIGHT_CLASS_BULKY
+	STR.max_combined_w_class = 20
+	STR.can_hold = typecacheof(list(
+	/obj/item/storage/pill_bottle,
+	/obj/item/reagent_containers/hypospray,
+	/obj/item/healthanalyzer,
+	/obj/item/reagent_containers/syringe,
+	/obj/item/clothing/glasses/hud/health,
+	/obj/item/hemostat,
+	/obj/item/scalpel,
+	/obj/item/retractor,
+	/obj/item/cautery,
+	/obj/item/surgical_drapes,
+	/obj/item/autosurgeon,
+	/obj/item/organ,
+	/obj/item/implant,
+	/obj/item/implantpad,
+	/obj/item/implantcase,
+	/obj/item/implanter,
+	/obj/item/circuitboard/computer/operating,
+	/obj/item/stack/sheet/mineral/silver,

```

Comment:
if it's an organ storage box, why can it hold things that aren't organs?

Name consistency.

Reply:
It can hold silver do to being used for surgery tables, same with the operating table. Meaning you only have to carry on you without the box 5 wire, 2 rods 1 wrench. and 5 metal

Reply:
Really shouldn't be able to store raw materials to build things in a box meant for refrigerated organs Especially when these items don't actually fit on anything for a belt slot.

It shouldn't be able to carry surgery tools or regular medical gear either. this is just power creep of medical and surgery belts being combined into one item.

Reply:
What are you talking about? It cant hold all surgery and it can only hold a few medical things.
It also cant fit in bags.

Reply:
it holds portable, arguably combat-capable surgery suites. the only thing it doesn't carry is a saw.

It doesn't need to carry these things at all. Only organs.

Reply:
Dosnt hold a drill. Dosnt hold any healing other then a hypo and pill bottle, dosnt fit in bag, cant hold sprays cant hold patches cant hold 9/10 healing things. Its apple is to hold organs and to do surgery on the go.

Reply:
so it's a surgery belt with the ability for people to build surgery tables where-ever they please and swap organs as well.

Reply:
Its a belt that can help easly do surgery and hold organs, It cant hold rods, it cant hold tools other then surgery and it cant hold wire/glass
You still need to carry the drill and or saw for surgerys. It also cant hold defibs or items that would be used as a replacement.

Reply:
why is it there to begin with, then? 

Why can it hold implants?

Why can it hold AN ENTIRE AUTOSURGEON?

Make it only hold organs and nothing else and this will be fine. Stop trying to fit in powergame feature creep

Reply:
Autosurgeons are small, implants are small/replace parts, it only takes larger items that thematically would fit, why cant it hold a saw/drill? To long
Why cant it hold tools? Not Thematic
Its main use is for people to do surgery well not needing to place every item in a boh or such. 
Not like its a real medical belt that can hold everthing ever, fit in bag and act as a smaller better box

Reply:
It cost a alot of mats and plastic, it is not even fucken powercreep.